### DISCUSSION TITLE
Normal battery health?

### DISCUSSION THREAD

**Discussion by hmouridsen** [teslamate-org/teslamate#4226] [Q&A]
My YSR LFP looks like this. I'm wondering why the capacity jumps like 4 kWh when changing tires from 19 to 20 inch (summer/winter).

Also it shows 64 kWh from new, but afaik the CATL battery pack has never been 64 - more like 62 and now software locked to 59-60 kWH ish.

Can anyone help me understand what's going on here. 11% degradation on less than 30.000km seems concerning to me, and I never leave the car on 100%

Thanks for great software

 ![image](https://github.com/user-attachments/assets/addad4a4-7279-47a6-bb29-654b6216f57c)

**Comment by ktsenvolt** [comment]
Have you selected LFP on Dashboards? Also, you can enter custom battery capacity when new ie. 62kwh.

**Reply by hmouridsen** [reply]
> Have you selected LFP on Dashboards? Also, you can enter custom battery capacity when new ie. 62kwh.

Yes LFP is selected

Im curious to see if capacity goes up again, when mounting 19 inch winter tires.

**Reply by MrJoki007** [reply]
I have the same problem

**Comment by ykii** [comment]
Exact same problem here. Changing winter tyres (19") to summer tyres (20") makes battery health capacity to drop by 5kWh. Seems like a bug to me?

**Comment by JakobLichterfeld** [comment]
improved by #4644
